openedx/features/partners/views.py

```diff
@@ -14,29 +13,33 @@
 from django.utils.translation import get_language
 from django.views.decorators.debug import sensitive_post_parameters
 from django.views.decorators.http import require_http_methods
-from edxmako.shortcuts import render_to_response
-from eventtracking import tracker
-from notification_prefs.views import enable_notifications
 from pytz import UTC
-from rest_framework import status
 
-from openedx.core.djangoapps.user_api.views import RegistrationView
+import analytics
+from edxmako.shortcuts import render_to_response
+from eventtracking import tracker
 from lms.djangoapps.onboarding.models import EmailPreference, Organization, PartnerNetwork, UserExtendedProfile
-from nodebb.helpers import update_nodebb_for_user_status, set_user_activation_status_on_nodebb
+from nodebb.helpers import set_user_activation_status_on_nodebb, update_nodebb_for_user_status
+from notification_prefs.views import enable_notifications
 from openedx.core.djangoapps.lang_pref import LANGUAGE_KEY
 from openedx.core.djangoapps.user_api.preferences import api as preferences_api
+from openedx.core.djangoapps.user_api.views import RegistrationView
 from openedx.core.djangoapps.user_authn.cookies import set_logged_in_cookies
 from openedx.core.djangoapps.user_authn.views.register import REGISTER_USER, record_registration_attributions
-from openedx.features.partners.helpers import auto_join_partner_community, get_partner_recommended_courses
+from openedx.features.partners.helpers import (
+    auto_join_partner_community,
+    get_partner_recommended_courses,
+    get_partner_registration_countries
+)
 from openedx.features.partners.models import PartnerUser
 from openedx.features.student_account.helpers import save_user_utm_info
 from philu_overrides.user_api.views import LoginSessionViewCustom
+from rest_framework import status
 from student.models import Registration, UserProfile
 from student.views import password_change_request_handler
 
 from . import constants as partner_constants
-from .forms import PartnerAccountCreationForm
-from .forms import PartnerResetPasswordForm
+from .forms import PartnerAccountCreationForm, PartnerResetPasswordForm
```

Comment:
please execute pylint, and format as per the suggestion.

Reply:
I actually ran `isort` on this, which did all this.